chipsec/helper/win/win32helper.py

```diff
@@ -353,6 +353,8 @@ def create(self, start_driver):
         if logger().VERBOSE: logger().log( "[helper] service control manager opened (handle = 0x%08x)" % hscm )
 
         driver_path = os.path.join( chipsec.file.get_main_dir(), "chipsec", "helper", "win", self.win_ver, DRIVER_FILE_NAME )
+        if not os.path.isfile( driver_path ):
+            driver_path = os.path.join("C:\\Windows\\System32\\drivers\\RwDrv.sys"
```

Comment:
I think it's a bad idea to add interdependence between the two helpers. I'd rather windows helper fail if it doesn't find required driver than attempting to load a driver from another helper.